blackbird_python/blackbird/listener.py

```diff
@@ -427,6 +446,28 @@ def exitStatement(self, ctx: blackbirdParser.StatementContext):
         else:
             self._program._operations.append(operation)
 
+    def enterForloop(self, ctx: blackbirdParser.ForloopContext):
+        self._infor = True
+
+    def exitForloop(self, ctx: blackbirdParser.ForloopContext):
+        self._infor = False
+
+        if ctx.INT():
+            _FORVAR = range(int(ctx.INT().getText()))
+        elif ctx.vallist():
+            _FORVAR = ctx.vallist().getText().split(',')
```

Comment:
Does the `vallist()` context assume the syntax `[1, 2, 3]` (e.g., do you have to use square brackets?)

Reply:
`vallist` is currently only the content of the list, i.e. `1, 2, 3`, so any brackets could in practice be used, although the for-loop only supports lists with square brackets.